cli/azd/internal/appdetect/testdata/java-multimodules/application/build.gradle

```diff
@@ -0,0 +1,23 @@
+plugins {
```

Comment:
I'm wondering for making sure the test setup reflects the current feature set, would you agree with removing the `*.gradle` files for now?

We'll add it back when we support gradle -- but this requires a core azd change so it'll take some time before we get there.

Reply:
Yes, will do